src/Main.java

```diff
@@ -1,5 +1,39 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        int[] pay = {234000, 345000, 125432, 342523, 745622};
+        System.out.println();
+        System.out.println("Task 1");
+        int sum = Arrays.stream(pay).sum();
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+        System.out.println();
+        System.out.println("Task 2");
+        int min = pay[0];
+        int max = pay[0];
+        {
+            int i = 0;
+            while (i < pay.length) {
+                if (pay[i] < min) {
+                    min = pay[i];
+                }
+                if (pay[i] > max) {
+                    max = pay[i];
+                }
+                i++;
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила " + min + " рублей. Максимальная сумма трат за неделю составила " + max + " рублей");
+        System.out.println();
+        System.out.println("Task 3");
+        float average = 4f;
+        System.out.println("Средняя сумма трат за месяц составила " + sum / average + " рублей");
```

Comment:
В массиве может быть либо 4 либо 5 элементов и делить всегда на 4 не совсем корректно, надо делить на длину массива
```suggestion
        System.out.println("Средняя сумма трат за месяц составила " + sum / pay.length + " рублей");
```